lib/ansible/modules/database/misc/redis.py

```diff
@@ -93,6 +93,14 @@
             - A redis config value.
         required: false
         default: null
+    config_write:
```

Comment:
eewww .... ... if there is no other way .. fine, but do add in more explicit terms when you say it saves the 'current config' that it MAY also include other changes that are not related to this task.